pkg/minikube/machine/client.go

```diff
@@ -183,7 +186,23 @@ func (api *LocalClient) Create(h *host.Host) error {
 	}{
 		{
 			"bootstrapping certificates",
-			func() error { return cert.BootstrapCertificates(h.AuthOptions()) },
+			func() error {
+				// Due to https://github.com/docker/machine/issues/3634, we need to have lock for certs generation, otherwise
+				// it will break Docker-Env test in parallel tesing.
```

Comment:
add the issue number on github 
and change wording to
"this lock is needed to run docker-env tests in parallel issuue #... "